07LessonSeven/homework/index.html

```diff
@@ -1,17 +1,53 @@
 <!DOCTYPE html>
-<html>
+<html class="clearfix">
 
   <head>
-    <title>NASA</title>
-    <link rel="stylesheet" href="css/style.css">
+    <title>Professional Portal</title>
+    <link rel="stylesheet" href="css/style.css" />
   </head>
 
-  <section id="imaginethat">
-<img =src"/>
-</img>
-  </section>
   <body>
+    <!--header color should be white until nav  -->
+    <header>
+      <div>
+        <h1>Professional Portal</h1>
+      </div>
+
+      <!--add background color in css for nav  -->
+      <nav class="nav_menu" class="clearfix">
+        <!--color of link text should be white  -->
+        <ul>
+          <li>Home</li>
+          <li>Companies</li>
+          <li>Positions</li>
+        </ul>
+      </nav>
+    </header>
+
+    <!--end of header  -->
+
+    <!-- First box...will have green background color -->
+    <main>
+      <section class="ggreen" class="clearfix" class="centerme">
+
+        <h2>  Hire an employee </h2>
+        <p>Vivamus congue rhoncus sem et placerat. Fusce nec nunc lobortis lorem ultrices facilisis. Ut dapibus blandit nunc, et consectetur dui.</p>
+
+        <button class="buttongreen">Sign up as company.</button>
+      </section>
 
-  </body>
 
-</html>
+      <!--End of left box  -->
+
+      <!--Right box begins..color will be red  -->
+
+      <section class="ddark" class="clearfix" class="centerme">
```

Comment:
remember, to have multiple classes, all we need to do is put a space between them. class="class1 class2 class_3"